packages/autoprofile/precompile/build-all-addons.js

```diff
@@ -32,12 +32,11 @@ const LIBC_FAMILIES = [GLIBC, MUSL];
 // Maintenance Note:
 // This should be kept in sync with native-dep-packs/rebuild-precompiled-addons.sh -> ABI_VERSIONS.
 const ABI_VERSIONS = {
-  // 14.17.x was no longer working
-  // https://stackoverflow.com/questions/76094428/debian-stretch-repositories-404-not-found
```

Comment:
Question: Why did you remove the comments?

Reply:
@kirrg001 ✋  I removed those. I felt they are no longer relevant. Seemed to more an explanation of why it was changed from 14.17. to 14.21. But 14.21. is the most recent 14.x version anyway, we wouldn't randomly go back to 14.17 anyway would we?

Reply:
Oke 🫡